server/routes.go

```
@@ -189,11 +189,12 @@ func GenerateHandler(c *gin.Context) {
 	// TODO: set this duration from the request if specified
```

I think we'll need to return an error as well for `/api/embeddings` ?